### DISCUSSION TITLE
404 Deployment error

### DISCUSSION THREAD

**Discussion by Kulaneian** [vercel/community#198] [Help] [answered]
I have a reactjs website and the repository is in gitlab. Every time I make a new commit in gitlab I go to vercel to see the deployment status. And every time I get an 404 deployment error. The deployment that is live right now on Vercel is a commit from 4 days ago. Every new commit I make redirects to a 404 error page. On gitlab's side everything works well, so the issue is on vercel's side.

Can someone explain what to do?

**Comment by Pieparker** [comment]
Firstly, you should check to see whether the issue occurs locally. Usually, this is because of a discrepancy between your output directory and where your output files live (you can check the latter by using the `Source` tab under your most recent deploy within the UI and selecting `Output`). Double-check this!

If you still need a hand, try sharing one of your deploys or the deploy ID.

**Reply by Kulaneian** [reply]
![afbeelding](https://user-images.githubusercontent.com/75037952/149926050-dd0cf9f7-4aac-4137-97ef-ab64ce814b6e.png)

I keep getting this screen with every new commit when I check the deployment to vercel........

**Comment by Pieparker** [comment, marked as answer]
Thanks! The URL is enough for me to identify the project. So you know, the project ID is the value at the end of the URL for a deployment, e.g. `https://vercel.com/[owner-name]/[project-name]/[deployment-id]`. So, the ID for this deploy is `42RuJKpKifzKCgDXpZx1XWype5Lp`.

According to your deployment overview – `https://vercel.com/[owner-name]/[project-name]/deployments` – `42RuJKpKifzKCgDXpZx1XWype5Lp` is the most recent deployment, made 22 hours ago.

You may be getting `404`s as ownership of the project has recently changed. You'll need to ensure that you are a team member, in order to view that page.

**Reply by Kulaneian** [reply]
Yeah I am the owner. I upgraded the plan after I couldnt fix the issue, so I could get help from an account manager. Anyway the deployment you see from yesterday is a commit from 4 days ago. The newest commits dont deploy to vercel somehow. Yesterday there was another commit I made and it did went well on gitlab. On gitlab it shows the commits are succesful, but on vercel I get a 404 error page.

**Reply by Kulaneian** [reply]
I clicked on redeploy on vercel multiple times and it keeps deploying the commit from 4 days ago.....

**Reply by Pieparker** [reply]
Redeploying on Vercel will redeploy **that** deployment, not the latest commit. If your Git commits aren't automatically triggering deployments, [check this out](https://vercel.com/support/articles/why-aren-t-commits-triggering-deployments-on-vercel)!
